core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -451,4 +468,43 @@ protected void startQuestDb(
     ) {
         workerPool.start(log);
     }
+
+    private static CharSequence getQuestDbVersion(final Attributes manifestAttributes) {
+        return manifestAttributes.getValue("Implementation-Version");
+    }
+
+    private static CharSequence getJdkVersion(final Attributes manifestAttributes) {
+        return manifestAttributes.getValue("Build-Jdk");
+    }
+
+    private static CharSequence getCommitHash(final Attributes manifestAttributes) {
+        return manifestAttributes.getValue("Build-Commit-Hash");
+    }
+
+    private static Map<CharSequence, CharSequence> buildInternalProperties() throws IOException {
```

Comment:
Delivering build information to server can be done via `BuildInformation` interface above, which configuration can keep the reference to.